### PR TITLE
Paas 2089

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-2089

Short description:

### CODE REVIEW

File: mixins/haproxy.yml [hunk 1]
```diff
@@ -194,3 +194,22 @@ actions:
     - environment.nodegroup.ApplyData [proc, cp]:
         # Formatting it like that is the only way I found to set a boolean and not a string (true and not "true")
         data: "{ basicAuthEnabled: ${this.enable_auth_basic} }"
+
+  disableHaProxyHealthcheck:
```
Comment (kevinvandel) [inline mixins/haproxy.yml:198]: I'm sorry but I can't tolerate this case for `HaProxy` :stuck_out_tongue: 
We use `Haproxy` almost everywhere else (I found 2 exceptions with `HAProxy`)

File: mixins/haproxy.yml [hunk 2]
```diff
@@ -194,3 +194,22 @@ actions:
...
+    cmd[bl]: |- # make haproxy check target /ping.jsp instead of /modules/healthcheck
+      haproxy_conf_file=/etc/haproxy/haproxy.cfg.jahia/jahia-cloud.cfg
+      if ! grep -qe "#option httpchk.*healthcheck" $haproxy_conf_file ; then
+        sed -i -e 's/\(option httpchk\)/#\1/g' -e 's/\(http-check expect rstring\)/#\1/g' $haproxy_conf_file;
+        sed -i -e '/\(#option httpchk\)/a option httpchk GET "/ping.jsp" "HTTP/1.0\\r\\nUser-Agent: HAProxy"' -e '/\(http-check expect rstring\)/a http-check expect rstring PONG' $haproxy_conf_file;
+        service haproxy reload
+      fi;
+    user: root
+
+  enableHaProxyHealthcheck:
```
Comment (kevinvandel) [inline mixins/haproxy.yml:208]:
```suggestion
  enableHaproxyHealthcheck:
```

**Review by kevinvandel — CHANGES_REQUESTED** [review]

File: mixins/haproxy.yml (hunk 2 above)
Comment (Ronan-Gorain) [reply mixins/haproxy.yml]: updated

**Review by laurentfufu — DISMISSED** [review]

File: mixins/haproxy.yml (hunk 2 above)
Comment (laurentfufu) [reply mixins/haproxy.yml]: for me it's more the `;` at the end of the line that shocks me :trollface:

**Review by kevinvandel — APPROVED** [review]

**Pull request merged**